### ISSUE TITLE
predefined_value @daily does not work

### ISSUE THREAD

**Issue by czchen** [chef-cookbooks/cron#52]
I use `predefined_value @daily` in `cron_d`, and it generates `* * * * *` in the cron file.

Recipes:

```
cron_d 'hello-world' do
    predefined_value @daily
    command 'echo hello world'
end
```

Result:

```
# Crontab for hello-world managed by Chef. Changes will be overwritten.

* * * * * root echo hello world
```

**Comment by alexandre-zia-movile** [comment]
I'm having same issue, using pre-defined value @hourly and it generates entry as: * * * * *

**Comment by alexderz** [comment]
Just did a quick test for all 8 predefined values, and the files are getting created with the correct content. Might you be using an older version? Check if the resources/d.rb lists predefined_values as a option.

**Comment by HerrStinson** [comment]
For future reference (since I ran into this myself), the predefined_value has to be enclosed by quotation marks, as it is a string:
```
cron_d 'hello-world' do
    predefined_value "@daily"
    command 'echo hello world'
end
```

**Comment by cheeseplus** [comment]
Closing as fixed by #56
